recipe/meta.yaml

```diff
@@ -38,11 +38,12 @@ requirements:
     - ipython
     - jinja2 >=2.1
     - jupyter_core
-    - jupyter_server >=1.4,<2
+    - jupyter_server >=1.16,<2
```

Comment:
`jupyter_server` version `1.16` does not seem to be supported in this case.
https://github.com/jupyterlab/jupyterlab/blob/v3.4.4/setup.cfg#L39

Reply:
`~=1.16` means "around 1.16"

Reply:
In this case it's equivalent to `>=1.16,==1.*`

Reply:
I suggest updating the pinning to be `jupyter_server >=1.16.0,<2.0.0a0` to make the bounds more explicit.